plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/font-pairing-variations/index.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+/* eslint-disable @woocommerce/dependency-group */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/**
+ * External dependencies
```

Comment:
Looks like `External dependencies` block is duplicated, is this intended?

Reply:
Nope. Removed in e20ab485533907ba87e853b97d45c1fc70f13927. 😄